derp/derp_server.go

```diff
@@ -913,6 +931,10 @@ type sclient struct {
 	// Owned by sender, not thread-safe.
 	bw *bufio.Writer
 
+	bytesSent *uint64    // accessed atomically
```

Comment:
I don't think we'll need these two fields if we go the route of reading `/proc/net/tcp6` instead with a new HTTP debug handler that streams JSON stats.